backend/app/views/spree/admin/shared/_locale_selection.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="admin-locale-selection">
+  <% available_locales = Spree.i18n_available_locales %>
+  <% if available_locales.size > 1 %>
+    <select class="js-locale-selection custom-select fullwidth">
+      <%=
+        options_for_select(
+          available_locales.map do |locale|
+            [I18n.t('spree.i18n.this_file_language', locale: locale, default: locale.to_s, fallback: false), locale]
```

Comment:
Have you considered using some more explicit string to mark locales available, like `active: true`? 
I know it's not a proper translation but how can a user easily understand that this key is what will make the translation file available?

Reply:
I think this is good for now and works with existing locales w/o any further changes.